src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public String getRandomBall() {
+        Random random = new Random();
```

Comment:
Think about which variables should be local in the method and which should be class-level

see https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery